src/js/helpers.ts

```diff
@@ -123,6 +123,20 @@ export function validateConsent(
 	return true;
 }
 
+/**
+ * Recieves a string (or array of stings) value that indicates if the consent was checked or not
+ * and converts to "status" (boolean) value
+ * Posible values: 'yes', 'no', ['yes', 'no']
+ * ACQ-2216: side effect when adding a hidden input field to cover unchecked checkbox
+ */
+function convertStringConsentToBoolean(value) {
+	if (Array.isArray(value)) {
+		return Array.from(value).includes('yes');
```

Comment:
Is `Array.from()` needed here? We already checked if it's an array? 
```suggestion
		return value.includes('yes');
```